scripts/curate_basel.py

```diff
@@ -0,0 +1,127 @@
+import os
+import shutil
+import json
+import argparse
+
+root_data = '/Users/alex/data/basel-mp2rage_raw'
+output_data = '/Users/alex/data/basel-mp2rage_raw_curated'
+
+
+def get_parameters():
+    parser = argparse.ArgumentParser(description='This script is curating dataset Delbono Wakehealth to BIDS')
+    parser.add_argument("-d", "--data",
+                        help="Path to folder containing the dataset to be curated",
+                        required=True)
+    parser.add_argument("-o", "--outputdata",
+                        help="Path to output folder",
+                        required=True,
+                        )
+    arguments = parser.parse_args()
+    return arguments
+
+
+def main(root_data, output_data):
+    if os.path.isdir(output_data):
+        shutil.rmtree(output_data)
+    os.makedirs(output_data)
+
+    dict_images = {
```

Comment:
Personally I find this more readable

```suggestion
    images = {
```